.github/workflows/main.yml

```diff
@@ -9,6 +9,7 @@ on:
 env:
   CARGO_TERM_COLOR: always
   RUST_CACHE_KEY: rust-cache-20240617
+  CARGO_PROFILE_DEV_DEBUG: none
```

Comment:
currently checking if that makes it less likely to exceed github's disk usage limits